ibis/tests/all/test_window.py

```diff
@@ -3,7 +3,8 @@
 
 import ibis
 import ibis.common.exceptions as com
-from ibis.tests.backends import Csv, OmniSciDB, Pandas, Parquet
+from ibis.tests.backends import Csv, Impala, OmniSciDB, Pandas, Parquet, \
```

Comment:
i think we generally prefer wrapping these in parentheses rather than `\`

Reply:
Got it